2.6/README.md

```diff
@@ -47,7 +47,7 @@ $ curl 127.0.0.1:8080
 Environment variables
 ---------------------
 
-To set these environment variables, you can place them as a key value pair into a `.s2i/environment`
+To set these environment variables, you can place them as a key value pair into a `.sti/environment`
```

Comment:
The previous version is the correct one here. Likely did not get into downstream yet.